src/Tooltip/Tooltip.js

```diff
@@ -4,19 +4,36 @@ import styled from 'styled-components';
 
 import { shadow } from '../common';
 
+const DownStyle = `
+  bottom: -4px;
+  transform: translate(-50%, 100%);
+`;
+const UpStyle = `
+  top: -4px;
+  transform: translate(-50%, -100%);
+`;
+const LeftStyle = `
+  transform: translate(-150%, 0);
+`;
+const RightStyle = `
+  transform: translate(50%, 0);
+`;
+
 const Tip = styled.span`
   position: absolute;
-  bottom: -4px;
   left: 50%;
   z-index: 1;
-  transform: translate(-50%, 100%);
   display: ${props => (props.show ? 'block' : 'none')};
   padding: 4px;
   border: 2px solid ${({ theme }) => theme.borderDarkest};
   background: ${({ theme }) => theme.tooltip};
   box-shadow: ${shadow};
   text-align: center;
   font-size: 1rem;
+  ${props => (props.direction === 'up' ? UpStyle : '')}
+  ${props => (props.direction === 'down' ? DownStyle : '')}
+  ${props => (props.direction === 'right' ? RightStyle : '')}
+  ${props => (props.direction === 'left' ? LeftStyle : '')}
```

Comment:
I think it would be cleaner to do something like:

```
const positioningStyles = {
    top: TopStyle,
    bottom: BottomStyle,
    left: LeftStyle,
    right: RightStyle
}
```

and then in styled component:

```
...
${props => positioningStyles[props.position]}
```